apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/LlmProviderApiKeyResourceTest.java

```diff
@@ -218,4 +194,15 @@ private void checkEncryption(UUID id, String workspaceId, String expectedApiKey)
         });
         assertThat(EncryptionUtils.decrypt(actualEncryptedApiKey)).isEqualTo(expectedApiKey);
     }
+
+    private void assertPage(Page<ProviderApiKey> actual, List<ProviderApiKey> expected) {
+        assertThat(actual.content()).hasSize(expected.size());
+        assertThat(actual.page()).isEqualTo(0);
+        assertThat(actual.total()).isEqualTo(expected.size());
+        assertThat(actual.size()).isEqualTo(expected.size());
+
+        for (int i = 0; i < expected.size(); i++) {
+            assertThat(actual.content().get(i).provider()).isEqualTo(expected.get(i).provider());
+        }
+    }
 }
```

Comment:
Missing an empty line at the end of file

Reply:
Done